battleship_sem.py

```diff
@@ -451,4 +453,4 @@ def battleship_game(play_count=0, won_count=0, lost_count=0):
     else:
         print(f"It was nice playing with you, {player1}! CU next time.")  
 
-battleship_game()        
```

Comment:
you broke the formatting, please check your formatter

Reply:
I'm so sorry. I'll fix it right away, boss!